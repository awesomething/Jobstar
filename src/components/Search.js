function Search() {
    return (
      <div>
        <h3>search page</h3>
      </div>
    );
  }
  export default Search;
  