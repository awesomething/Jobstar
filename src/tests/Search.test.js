import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Search from '../components/Search'



describe(`Search component`, () => {
  const props = {
    id: 'a',
    name: 'test-class-name',
    
  }

  it('renders a .Search by default', () => {
    const wrapper = shallow(<Search />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })

  it('renders the Search given props', () => {
    const wrapper = shallow(<Search {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
