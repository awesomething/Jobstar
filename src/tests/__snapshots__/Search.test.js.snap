// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component renders a .Search by default 1`] = `
<div>
  <h3>
    search page
  </h3>
</div>
`;

exports[`Search component renders the Search given props 1`] = `
<div>
  <h3>
    search page
  </h3>
</div>
`;
